### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by csuich2** [taligentx/dscKeybusInterface#138] [dscKeybusInterface Discussion]
Any thoughts of when you might publish the next release? I see there are some features in develop that have been in since last year.

**Comment by Gysbertbotes** [comment]
Good day. i have a DSC PC1864 panel with a DSC RFK5500 LCD keypad.
I have a Arduino nano 33 iot that i would like to use on OPENHAB2.
any new releases i can use that supports wifi and not ethernet.
